x-pack/platform/plugins/shared/index_management/public/application/sections/home/index_list/index_actions_context_menu/index_actions_context_menu.js

```diff
@@ -20,11 +20,19 @@ import {
   EuiSpacer,
   EuiConfirmModal,
   htmlIdGenerator,
+  EuiText,
 } from '@elastic/eui';
 
+import { ConvertToLookupIndexModalContainer } from '../details_page/convert_to_lookup_index_modal/convert_to_lookup_index_modal_container';
```

Comment:
How much trouble would it be to convert this file to typescript? There are very few js files left in our code base and they're always high priority to convert to ts. Might be best done as a separate PR.